src/Microsoft.AspNetCore.Razor.Tools/Application.cs

```diff
@@ -27,10 +35,10 @@ public Application(CancellationToken cancellationToken, ExtensionAssemblyLoader
 
             HelpOption("-?|-h|--help");
 
-            Commands.Add(new ServerCommand(this));
-            Commands.Add(new ShutdownCommand(this));
-            Commands.Add(new DiscoverCommand(this));
-            Commands.Add(new GenerateCommand(this));
+            Commands.Add(new ServerCommand(this, Out, Error));
+            Commands.Add(new ShutdownCommand(this, Out, Error));
+            Commands.Add(new DiscoverCommand(this, Out, Error));
```

Comment:
lol

Reply:
I wonder if we should just fix this in the source for `CommandLineApplication`. Seems like a common concern.

Reply:
This definitely looks like a common problem. It's atleast worth filing a bug.

Reply:
Filed https://github.com/aspnet/Home/issues/2963